content/en/mobile_testing/mobile_app_tests/steps.md

```diff
@@ -0,0 +1,170 @@
+
+{{< img src="mobile_testing/injecting_variable.png" alt="Type Text step to use variables in mobile tests" style="width:25%" >}}
+
+To use a variable on automatically recorded steps, add a step name and specify the variable value to input the variable value while recording. 
+
+#### Scroll
+
+Mobile app tests automatically scroll to the elements that need to be interacted with. In most cases, you do not need to add a scroll step manually. Use the scroll step when you need to trigger an additional interaction, such as an infinite scroll.
+
+Specify the number of pixels you want the mobile app test to scroll vertically and horizontally.
+
+{{< img src="mobile_testing/scroll_step.png" alt="Scroll step in a mobile test recording" style="width:60%;" >}}
+
+By default, the **Scroll** step scrolls through the entire page. If you need to scroll on a specific element (for example, a specific `<div>`), click **Starting Element** and select an element you want the mobile app test to scroll on.
+
+#### Press back
+
+Interacting with the **Back** button below the mobile application records a step.
+
+{{< img src="mobile_testing/press_back.mp4" alt="Recording a Press Back step in a mobile test" video=true >}}
+
+#### Wait
+
+If you know that a page or page element takes more than 60 seconds to load, you can add a wait step with a max value of 300 seconds.
+
+{{< img src="mobile_testing/wait_step.png" alt="Recording a Wait step in a mobile test" style="width:60%;" >}}
+
+By default, mobile app tests wait for a page to be fully loaded before performing a step or the next step with a timeout of 60 seconds. This additional time is systematically added to **every run** of your mobile app test's recording.
+
+#### Rotate device
+
+Add a name to the step and select **Portrait** or **Landscape** mode.
+
+{{< img src="mobile_testing/rotate_device.png" alt="Recording a Rotate Device step in a mobile test" style="width:60%" >}}
+
+#### Open deep link
+
+Add a name to the step and enter a deep link URI.
+
+{{< img src="mobile_testing/open_deep_link.png" alt="Recording an Open Deep Link step in a mobile test" style="width:60%" >}}
+
+For more information about additional configuration in test steps, see [Advanced Options for Mobile App Test Steps][4].
+
+### Subtests
+
+You can run mobile app tests within other mobile app tests to reuse existing workflows up to two levels of nesting.
+
+To use an existing mobile app test as a subtest, click **Add New Subtest**, select a mobile app test from the dropdown menu, and click **Add Subtest**.
+
+To convert steps from your current mobile app test into a subtest, click on the **Extract From Steps** tab, select the recorded steps you want to extract, and click **Convert to Subtest**. By default, a subtest executes in sequence with the previous steps of the parent test. 
+
+{{< img src="synthetics/browser_tests/advanced_options/subtest.png" alt="Add a subtest in a browser test" style="width:60%;" >}}
```

Comment:
We'll need to update this once the feature is rolled out because this screenshot is for Browser Testing

Reply:
Reminder to update this screenshot!